ci-operator/step-registry/ipi/conf/aws/custom-vpc/ipi-conf-aws-custom-vpc-commands.sh

```diff
@@ -9,30 +9,36 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
-
 # subnet and AZs
-all_subnet_ids="${SHARED_DIR}/all_subnet_ids"
-availability_zones="${SHARED_DIR}/availability_zones"
-if [ ! -f "${all_subnet_ids}" ] || [ ! -f "${availability_zones}" ]; then
-    echo "File ${all_subnet_ids} or ${availability_zones} does not exist."
-    exit 1
+# for C2S and SC2S, provide private zone and AZ only
+if [ X"${CLUSTER_TYPE}" == X"aws-c2s" ] || [ X"${CLUSTER_TYPE}" == X"aws-sc2s" ]; then
+  subnet_ids_file="${SHARED_DIR}/private_subnet_ids"
+  az_file="${SHARED_DIR}/private_availability_zones"
+else
+  subnet_ids_file="${SHARED_DIR}/all_subnet_ids"
+  az_file="${SHARED_DIR}/availability_zones"
```

Comment:
Can the SC2S/C2S workflows overwrite `all_subnet_ids` (with `private_subnet_ids`) and `availability_zones` with `private_availability_zones` in an earlier step? That would avoid the need for this conditional.

Reply:
Updated.